projects/1.20/assets/tacz-plus-timeless-and-classics-feature-addon/taczplus/lang/zh_cn.json

```
@@ -0,0 +1,20 @@
+{
+    "key.taczplus.deadeye": "死神之眼",
+    "subtitles.deadeyebeat": "死神之眼",
+    "key.taczplus.focus": "屏息",
+    "subtitles.deadeye": "死神之眼",
+    "key.taczplus.lean_left": "左探头",
+    "subtitles.exitdeadeye": "死神之眼退出",

```

```suggestion
    "subtitles.exitdeadeye": "死神之眼：退出",
```
字幕格式参考原版
此类问题请全文通查并修改

---

改了